### DISCUSSION TITLE
Is there any way to control the size of CloseButton in Drawer?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#735] [Q&A] [answered]
It appears that Drawer is using ActionIcon for CloseButton. Is there any way to control its size to make it larger?

**Comment by rtivital** [comment, marked as answer]
You you can customize CloseButton styles with `closeButton` selector using Styles API

**Reply by steiner-io** [reply]
How exactly did you achieve this? I was able to adjust the size of the button, however I was not able to adjust the size of the underlying icon.

**Reply by rtivital** [reply]
You can target svg element inside `closeButton`, for example

```css
.closeButton svg { width: 50px; height: 50px }
```

You can also hide close button with `withCloseButton={false}` prop and render your custom button instead

**Reply by steiner-io** [reply]
Thanks rtivital. The only way was able to achieve this was through a css file and I used the following:

`.mantine-Drawer-closeButton svg {
    width: 2rem;
    height: 2rem;
}`

Is there no inline CSS approach for it?

**Reply by rtivital** [reply]
You can add these styles via `styles` prop, I've shared styles that you need to apply

**Reply by steiner-io** [reply]
How do you apply styles to via desctendent combinator, e.g in this case .closeButton svg?

The following is not valid code:
`    styles={
            {
              .closeButton svg { width: 50px; height: 50px }
            }
          }`


My drawer component looks currently like this:
`      <Drawer
          opened={opened}
          onClose={() => setOpened(false)}
          title="Filter settings"
          padding="xl"
          size="xl"
        >
          //<RangeForm onSubmit={closeDrawer} />
        </Drawer>`

**Reply by rtivital** [reply]
`<Drawer styles={{ closeButton: { '& svg': { width: 50, height: 50 } } }} />`

**Reply by steiner-io** [reply]
@rtivital Thank you very much. '& svg' was the missing piece.

**Reply by Strajk** [reply]
For future visitors from Google, nested selectors in styles prop (`{ '& svg':`) is no longer supported in Mantine 7 https://mantine.dev/guides/6x-to-7x/#styles-prop
